WooCommerce/Classes/Analytics/WooAnalyticsEvent+CustomFields.swift

```diff
@@ -0,0 +1,138 @@
+import Foundation
+import enum Networking.MetaDataType
+
+extension WooAnalyticsEvent {
+    enum CustomFields {
+        /// Event property keys
+        private enum Keys {
+            static let type = "type"
+            static let fieldsCount = "field_count"
+            static let fieldsSize = "fields_size"
+            static let hasJsonFields = "has_json_fields"
+            static let isJson = "is_json"
+            static let has_html = "has_html"
+            static let editedFieldCount = "edited_field_count"
+            static let addedFieldCount = "added_field_count"
+            static let deletedFieldCount = "deleted_field_count"
```

Comment:
Regarding this, I just noticed a typo I did in Android, the plan follows what you have here, but in Android I [used](https://github.com/woocommerce/woocommerce-android/blob/591a59ce47edba00653cd472265de23d741e8a0b/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/customfields/list/CustomFieldsViewModel.kt#L242-L244) `edited_fields_count` (it's a mess in Android 🤦, `added_field_count` is different)

What do you prefer here, should we update the plan to add an `s` to all keys, meaning to use `edited_fields_count` or should I update Android? I think adding the `s` is better as it's also aligned with the other keys above.

Reply:
Since Android is already gathering data, I think it's better to follow it for consistency 👍🏼